zoompan.py

```diff
@@ -0,0 +1,160 @@
+"""
```

Comment:
The coding style from this file looks very different, where does it comes from?

Reply:
Found it: https://gist.github.com/t20100/e5a9ba1196101e618883

If you use someone else's code, you need to keep the licence linked to it. Please add back the copyright header and a link to the original code

Reply:
Addressed in: bde44402c18f1f0cacd51f126910895c40c70f7e